pyccel/ast/numpyext.py

```diff
@@ -1090,7 +1090,7 @@ def _set_dtype_precision(self, x):
     def is_elemental(self):
         return False
 
-class NumpyMax(NumpyUfuncUnary):
+class NumpyAmax(NumpyUfuncUnary):
```

Comment:
I am more than happy for us to keep these class names. I agree with you that they are clearer